geode-core/src/main/java/org/apache/geode/internal/cache/TXManagerImpl.java

```diff
@@ -340,6 +341,9 @@ public void begin() {
       proxy = new TXStateProxyImpl(cache, this, id, null);
     }
     setTXState(proxy);
+    if (logger.isDebugEnabled()) {
+      logger.debug("begin tx: " + proxy);
```

Comment:
Change log statements to use this syntax:
```
logger.debug("begin tx: {}", proxy);
```

Reply:
Will make the change.